core/controllers/collection_editor.py

```diff
@@ -197,8 +219,33 @@ def put(self, collection_id):
 
         # TODO(bhenning): Implement other rights changes here.
         is_public = self.payload.get('is_public')
-
-        if is_public is not None:
+        is_publicized = self.payload.get('is_publicized')
+        new_member_username = self.payload.get('new_member_username')
+        new_member_role = self.payload.get('new_member_role')
+        viewable_if_private = self.payload.get('viewable_if_private')
```

Comment:
Please drop this option -- we don't support it for collections, and are phasing it out for explorations (it makes permissions too tricky).